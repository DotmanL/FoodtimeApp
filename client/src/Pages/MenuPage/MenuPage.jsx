import React from 'react';
import styled from 'styled-components';
import Menu from '../../components/Menu/Menu';

const Container = styled.div`
  position: relative;
  min-height: 100vh;
  background-color: #e5e5e5;
  padding-bottom: 5px;
  width: 100vw;
  @media screen and (max-width: 800px) {
  }
`;

const MenuPage = () => {
  return (
    <Container>
      <Menu />
    </Container>
  );
};

export default MenuPage;
