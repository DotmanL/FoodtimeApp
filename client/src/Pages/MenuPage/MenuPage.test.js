import React from 'react';
import { shallow } from 'enzyme';
import toJson from "enzyme-to-json";
import MenuPage from './Menupage'


it(' expect to return MenuPage Component', () => {

  const wrapper = shallow(<MenuPage />)
  expect(
    toJson(wrapper)
  ).toMatchSnapshot()
})

