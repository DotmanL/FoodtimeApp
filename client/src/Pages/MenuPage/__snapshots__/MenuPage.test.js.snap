// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` expect to return MenuPage Component 1`] = `
<styled.div>
  <Connect(Menu) />
</styled.div>
`;
